src/realm/realmActions.js

```diff
@@ -123,12 +123,12 @@ export const fetchRestOfInitialData = (): Action => async (
   if (auth.apiKey !== '' && pushToken === '') {
     refreshNotificationToken();
   }
+  dispatch(trySendMessages());
```

Comment:
While doing doInitialFetch the state.app.eventQueue is null hence the event which triggers EVENT_NEW_MESSAGE was not being received